todo-app/src/components/UpdateTodo.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+
+export default class UpdateTodo extends React.Component {
+
+    state = {
+        description: this.props.todo.description,
```

Comment:
What made you decide to keep `editMode` in your MobX store, but keep the actual editing data in local state? Could you defend this decision in a code review?